### ISSUE TITLE
[SUGGESTION] For walking Around moving ship

### ISSUE THREAD

**Issue by surajnairpv** [tsunamayo/Starship-EVO#5452]
Just a few thoughts I had in mind.
 

1) Is it possible to add an option where we can switch off the dampeners in space allowing the ship to drift in the direction we want?
This would be very useful to preserve fuel as well as move around the ship while its still drifting/moving in space.

2) Can we have the autopilot kept on when we get off the seat controls? that would really be awesome.

I do realize the game is still being developed but thought I would share this across.
Thanks 😄 👍

**Comment by COMICLER** [comment]
this would be awesome to have

**Comment by tsunamayo** [comment]
@surajnairpv Hi,
so for 1) I am against, it seems unfun to require the player to do this, plus you only use fuel in lightcruise. It is not a realistic game.
2) Yes this should work already so I might need to fix it.
Thanks

**Comment by Majikmonster** [comment]
You can turn off your thrusters manually by connecting them to a switch or other logic. You get a running start and then switch them off. You'll very slowly bleed off speed and eventually stop, so it's not foolproof, but it works well enough.

It would be useful to have an autopilot option that didn't use lightcruise. Something along the lines of "hold course", keeping your current velocity like cruise control for cars. Or maybe the ship would plot a path forward and do course corrections around obstacles in the way of the flight path so you can autopilot through an asteroid ring or a crowded station/port.

Aligning to a natural gravity field would be nice, too. Letting your autopilot fly an orbit around a planet while you walk around on it or take a shuttle down and meet the ship at another point along its path later on would be fantastic.

**Comment by surajnairpv** [comment]
@Majikmonster I agree with you on this, autopilot without light cruise would really make things interesting..
